content/rs/administering/database-operations/exporting-data.md

```diff
@@ -5,11 +5,33 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-Using the Export option, you can back up a database at any time to an
-FTP server, Amazon S3, or OpenStack Object Storage ("Swift"). Other
-cloud storage options, such as Azure Geo-Redundant Storage, SoftLayer
-Object Storage, and Google Cloud Storage will be added in a future
-release. You cannot back up to the local machine.
+You can [schedule backups]({{< relref "/rs/administering/database-operations/exporting-data.md" >}})
```

Comment:
1) Please update the link. I think it's incorrect in this context. 
2) Apply the updates from /rs/administering/database-operations/database-backup.md to this page as well

Reply:
Link looks OK to me. I'll apply to the other page.